configs/dynamic_voxelization/metafile.yml

```diff
@@ -0,0 +1,50 @@
+Collections:
+  - Name: Dynamic Voxelization
+    Metadata:
+      Training Data: KITTI
+      Training Techniques:
+        - AdamW
+      Training Resources: 8x V100 GPUs
+      Architecture:
+        - SECOND
+        - Dynamic Voxelization
+        - Focal Loss
```

Comment:
Is focal loss also used in other models?

Reply:
Whether the loss should be regarded as a part of architecture?

Reply:
Can omit Focal Loss and SECOND.

Reply:
Yes, sir. Please refer to metafile in fp16, free_anchor, imvotenet, mvxnet, parta2, pointpillars, regnet, second and ssn. Due to the high frequency, I thought focal loss is a common architecture and is written into metafile.

Reply:
Thanks for your advice. I have updated the metafiles.